pages/utils/sum.ts

```diff
@@ -0,0 +1,8 @@
+export function sum(a: number, b: number, format: "html" | "text" = "html") {
```

Comment:
Maybe it's best to rename the function to "getSum" or calculateSum" in order to know that this function calculates the sum

Reply:
OK. I think this aligns with our coding standards / style guide.